c10/util/C++17.h

```diff
@@ -275,21 +275,21 @@ decltype(auto) if_constexpr(ThenCallback&& thenCallback, ElseCallback&& elseCall
   // This will give us better error messages.
   if constexpr(Condition) {
     if constexpr (detail::function_takes_identity_argument<ThenCallback>::value) {
-      return std::forward<ThenCallback>(thenCallback)(detail::_identity());
+      return ::std::forward<ThenCallback>(thenCallback)(detail::_identity());
     } else {
-      return std::forward<ThenCallback>(thenCallback)();
+      return ::std::forward<ThenCallback>(thenCallback)();
     }
   } else {
     if constexpr (detail::function_takes_identity_argument<ElseCallback>::value) {
-      return std::forward<ElseCallback>(elseCallback)(detail::_identity());
+      return ::std::forward<ElseCallback>(elseCallback)(detail::_identity());
     } else {
-      return std::forward<ElseCallback>(elseCallback)();
+      return ::std::forward<ElseCallback>(elseCallback)();
     }
   }
 #else
   // C++14 implementation of if constexpr
-  return detail::_if_constexpr<Condition>::call(std::forward<ThenCallback>(thenCallback),
-                                                 std::forward<ElseCallback>(elseCallback));
+  return detail::_if_constexpr<Condition>::call(::std::forward<ThenCallback>(thenCallback),
```

Comment:
Would line 310 need the same disambiguation also?

Reply:
Fixed. Thanks!